content/webapp/components/EventSchedule/EventScheduleItem.tsx

```diff
@@ -70,20 +70,19 @@ const EventScheduleItem: FunctionComponent<Props> = ({
     <GridWrapper>
       <div className="grid">
         <EventTimesWrapper>
-          {event.times &&
```

Comment:
If you look at the types, `event.times` is always defined – the only change here is removing the unnecessary conditional.